src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,48 @@
-// your components will all go in this `component` directory.
-// feel free to change this component.js into TodoList.js
+import React from 'react';
+// import Todo from './Todo'
+
+class TodoList extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      input: "",
+      list: [],
+    }
+  }
+
+  change(event){
```

Comment:
```suggestion
handleChange = e => {
		const { type, name, value } = e.target;
		const val = type === 'number' ? parseFloat(value) : value;
		this.setState({ [name]: val });
	};
```

Reply:
a more verbose handleChange method that also converts any number represented as a string to an actual number.